src/pages/MuitoManga/main.ts

```diff
@@ -0,0 +1,79 @@
+import { pageInterface } from '../pageInterface';
+
+export const MuitoManga: pageInterface = {
+  name: 'MuitoMangá',
+  domain: 'https://muitomanga.com',
+  languages: ['Portuguese'],
+  type: 'manga',
+  isSyncPage(url) {
+    return utils.urlPart(url, 3) === 'ler' && utils.urlPart(url, 4) !== '';
+  },
+  isOverviewPage(url) {
+    return utils.urlPart(url, 3) === 'manga' && utils.urlPart(url, 4) !== '';
+  },
+  getImage() {
+    return j.$('.capaMangaInfo img').first().attr('data-src');
+  },
+  sync: {
+    getTitle(url) {
+      return j
+        .$('.widget-title')
+        .first()
+        .text()
+        .match(/Ler (.+) Capítulo/)!
+        .pop()!;
```

Comment:
Throws an error for me. I imagine it is because of the missing document ready in the init function

Ex.: https://github.com/MALSync/MALSync/blob/master/src/pages/MangaJar/main.ts#L87
![image](https://user-images.githubusercontent.com/12599334/165245678-94003e0c-7972-46e3-8c3d-0f3ec8e39c52.png)

https://muitomanga.com/ler/is-this-warrior-real/capitulo-21

Reply:
Can't repro here. But I'm going to add it :+1: